dockerfiles/alpine/obam/Dockerfile

```diff
@@ -90,7 +90,7 @@ RUN if [ -n "$WSO2_OB_KEYSTORES_URL" ]; then wget -O private-keys.jks "${WSO2_OB
 # read deployment.toml file
 ARG DEPLOYMENT_TOML_FILE=${WSO2_OB_ACCELERATOR_DIR}/repository/resources/deployment.toml
 RUN \
-    cp ${WSO2_OB_ACCELERATOR_DIR}/repository/resources/wso2am-4.0.0-deployment.toml ${DEPLOYMENT_TOML_FILE} \
+    cp ${WSO2_OB_ACCELERATOR_DIR}/repository/resources/${WSO2_SERVER}-deployment.toml ${DEPLOYMENT_TOML_FILE} \
```

Comment:
We need to change this in ubuntu docker file as well.
https://github.com/wso2/docker-open-banking/blob/master/dockerfiles/ubuntu/obam/Dockerfile#L99

Reply:
Fixed in ca0ee9826b522a84aa1d5609304231b11dd68d02